scripts/extract_from_ISO.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+#
+# Copyright (C) 2019 onwards, Samveen S. Gulati
+#
+# Script to extract FL1 or FL2 files from Lenovo BIOS update ISO image
+#
+
+# EL TORITO offset count (thanks to Hamish Coleman)
+FAT_OFFSET=71680
```

Comment:
Just to be sure you are aware - this number has been known to change, and is an artifact of the Lenovo build process (and not guaranteed).  There is a way to calculate it, if needed